moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -67,7 +75,8 @@ bool pilz_industrial_motion_planner::computePoseIK(const planning_scene::Plannin
     return false;
   }
 
-  robot_state::RobotState rstate(robot_model);
+  moveit::core::RobotState rstate = scene->getCurrentState();
+  
```

Comment:
Why this? Next line, you reset joint values to defaults.

Reply:
The only reason for using a robot state from the scene is that it comes with the attached objects / subframes, which the previous `robot_state::RobotState rstate(robot_model);` did not. No reason for choosing the current state in particular. If there's a neater way doing this I'm happy to change it

Reply:
Given that this state is used for IK, I suggest to drop `setToDefaultValues()`.

Reply:
Done :+1: